lib/ability.rb

```diff
@@ -60,6 +60,7 @@ def initialize(user, resource, controller = nil)
       can [:send_receipt, :show], Order
       if resource == Facility.cross_facility
         can :manage, [Reservation, User]
+        cannot [:edit, :update], Reservation
```

Comment:
jhanggi 8 minutes ago Member Pending
I'm not really loving this tendency towards using cannot. I think I'd rather whitelist the actions the user can do (so can [:read, :create], Reservation (not sure if there are other ones we need to add). It's probably a little more testing, but if we have decent ability specs, it should be fine.